### PR TITLE
Basic view controller & routes for products

### PR DESCRIPTION
**Pull request by YCHQR**
Finished the Basic views, controller and routes for product. We need to discuss how everything needs to be displayed. We can review together if y'all want

### CODE REVIEW

**Comment by YCHQR** [comment]
I'm sorry I forgot to make a pull request yesterday but I was able to push to this branch yesterday

File: app/views/pages/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <div class="row justify-content-center">
+    <div class="col-md-8">
+      <h1>Post a new product</h1>
+
+      <%= simple_form_for @product do |f| %>
+        <%= f.input :title %>
+        <%= f.input :description %>
+        <%= f.input :price_per_day %>
+        <%= f.button :submit, class: 'btn btn-primary' %>
+      <% end %>
```
Comment (piranha73) [inline app/views/pages/new.html.erb:14]: We need also a brand and a category selection to create a product

File: app/views/pages/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <% @products.each do |product| %>
+      <div class="card-product">
+        <div class="card" style="background-image: linear-gradient(rgba(0,0,0,0.3), rgba(0,0,0,0.2)), url('https://source.unsplash.com/500x400/?cocktail');">
+          <div class="card-product-infos">
+            <h2><%= product.name %></h2>
+          </div>
+          <%= link_to "", product_path(product), class: 'btn btn-flat' %>
+        </div>
+      </div>
+    <% end %>
```
Comment (piranha73) [inline app/views/pages/index.html.erb:15]: We don't need a cocktail actually, please remove it, we'll add the image later

File: app/views/pages/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<h1>Product Page</h1>
+<p>This is the Product page</p>
+
+<div class="banner" style="background-image: linear-gradient(-225deg, rgba(255, 99, 71, 0.2) 0%, rgba(255, 99, 71, 0.6) 50%);">
+  <div class="container">
+    <h1><%= @product.name %></h1>
+    <p>Checkout this great product!</p>
+    <%= link_to "Back to Homepage", root_path, class: "btn btn-ghost" %>
+  </div>
+</div>
```
Comment (piranha73) [inline app/views/pages/show.html.erb:10]: This is the show page of the Product so it has to go here ->
app/views/products/index.html.erb

File: app/views/pages/new.html.erb (hunk 1 above)
Comment (piranha73) [reply app/views/pages/new.html.erb:14]: Also, this file is in the wrong directory, has to be here -> app/views/products/new.html.erb

File: app/views/pages/index.html.erb (hunk 2 above)
Comment (piranha73) [reply app/views/pages/index.html.erb:15]: Even this file is in the wrong directory, should be here -> app/views/products/index.html.erb

**Review by piranha73 — COMMENTED** [review]
Today, add also the other action in the controller

**Review by piranha73 — APPROVED** [review]
Everything working fine! we'll add frontend

**Pull request merged**